src/Regulator.cpp

```diff
@@ -275,50 +382,36 @@ void Regulator::setFPPratio(int len)
         // use mMsecTolerance to set headroom
         if (mMsecTolerance == -500.0) {
             mAutoHeadroom = -1;
-            qDebug() << "PLC is in auto mode and has been set with variable headroom";
+            cout << "PLC is in auto mode and has been set with variable headroom \n";
         } else {
             mAutoHeadroom = -mMsecTolerance;
-            qDebug() << "PLC is in auto mode and has been set with" << mAutoHeadroom
-                     << "ms headroom";
+            cout << "PLC is in auto mode and has been set with" << mAutoHeadroom
+                 << "ms headroom \n";
             if (mAutoHeadroom > 50.0)
-                qDebug() << "That's a very large value and should be less than, "
-                            "for example, 50ms";
+                cout << "That's a very large value and should be less than, "
+                        "for example, 50ms \n";
         }
         // found an interesting relationship between mPeerFPP and initial
         // mMsecTolerance mPeerFPP*0.5 is pretty good though that's an oddball
         // conversion of bufsize directly to msec
         mMsecTolerance = (mPeerFPP * AutoInitValFactor);
     } else {
-        qDebug() << "PLC is using a fixed tolerance of " << mMsecTolerance << "ms";
+        cout << "PLC is using a fixed tolerance of " << mMsecTolerance << "ms \n";
     }
 
-    mHist = HIST;  //    HIST (default) is 4
-                   //    as FPP decreases the rate of PLC triggers potentially goes up
-                   //    and load increases so don't use an inverse relation
-
-    //    crossfaded prediction is a full packet ahead of predicted
-    //    packet, so the size of mPrediction needs to account for 2 full packets (2*FPP)
-    //    but trainSamps = (HIST * FPP) and mPrediction.resize(trainSamps - 1, 0.0) so if
-    //    hist = 2, then it exceeds the size
-
-    if (((mNumChannels > 1) && (mPeerFPP > 64)) || (mPeerFPP > 128))
-        mHist = 3;  // min packets for prediction, needs at least 3
-    if (gVerboseFlag)
-        cout << "mHist = " << mHist << " at " << mPeerFPP << "\n";
-
-    mXfrBuffer       = new int8_t[mPeerBytes];
-    mBroadcastBuffer = new int8_t[mPeerBytes];
+    mXfrBuffer = new int8_t[mPeerBytes];
     memset(mXfrBuffer, 0, mPeerBytes);
+    mBroadcastBuffer = new int8_t[mPeerBytes];
     memset(mBroadcastBuffer, 0, mPeerBytes);
-    mPacketCnt = 0;  // burg initialization
+
     mFadeUp.resize(mPeerFPP, 0.0);
     mFadeDown.resize(mPeerFPP, 0.0);
     for (int i = 0; i < mPeerFPP; i++) {
         mFadeUp[i]   = (double)i / (double)mPeerFPP;
         mFadeDown[i] = 1.0 - mFadeUp[i];
     }
-    mLastWasGlitch = false;
-    mNumSlots      = NumSlotsMax;
+
+    mNumSlots = NumSlotsMax;
```

Comment:
seems like we can get rid of the variable `mNumSlots` now that it's effectively a constant equal to `NumSlotsMax`

Reply:
that was a placeholder but let's do